pkg/kube/controllers/waitservice/pod_mutator.go

```diff
@@ -0,0 +1,130 @@
+package waitservice
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"code.cloudfoundry.org/quarks-operator/pkg/kube/apis"
+	"code.cloudfoundry.org/quarks-operator/pkg/kube/util/operatorimage"
+	"go.uber.org/zap"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/inject"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	"code.cloudfoundry.org/quarks-utils/pkg/config"
+)
+
+var (
+	// WaitKey is the key for identifying which service the pod has to wait for
+	WaitKey = fmt.Sprintf("%s/wait-for", apis.GroupName)
+)
```

Comment:
```suggestion
const (
	// WaitKey is the key for identifying which service the pod has to wait for.
	WaitKey = apis.GroupName + "/wait-for"
)
```

Reply:
Heh I tried to not do that to not break the code style of the project (see all the other controllers)